keyboards/ergodox_ez/keymaps/lukaus/keymap.c

```diff
@@ -0,0 +1,878 @@
+#include "ergodox_ez.h"
+#include "debug.h"
+#include "action_layer.h"
+#include "version.h"
+
+#include "keymap_german.h"
+
+#include "keymap_nordic.h"
+
+enum custom_keycodes {
+  PLACEHOLDER = SAFE_RANGE, // can always be here
+  EPRM,
+  VRSN,
+
+  // Programmer's Dvorak "macros" : 
+  //  To be paired with get_mods to enable both
+  //  Shift functionality and Programmer's Dvorak
+
+  DVP_ESC, // Grave escape basically i think
+  DVP_AMPR,
+  DVP_LBRACKET,
+  DVP_LCBR,
+  DVP_RCBR,
+  DVP_LPRN,
+  DVP_AT,
+
+  DVP_EQUAL,
+  DVP_ASTERISK,
+  DVP_RPRN,
+  DVP_PLUS,
+  DVP_RBRACKET,
+  DVP_EXLM,
+  DVP_HASH,
+
+  RU_2,
+  RU_3,
+  RU_4,
+  RU_6,
+  RU_7,
+  RU_DOT,
+
+  SHFT_COMMA,
+  SHFT_DOT,
+
+  RGB_SLD,
+  RGB_FF0000,
+  RGB_008000,
+  RGB_0000FF,
+  RGB_FFFFFF,
+  RGB_800080
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  // Programmer's Dvorak layer
+  [0] = KEYMAP(
```

Comment:
Please change all instances of `KEYMAP` to `LAYOUT_ergodox` (case-sensitive).

Here, and on Lines 73. 90, 108, 127 and 147.